build_defs.bzl

```diff
@@ -0,0 +1,229 @@
+# Description:
+#   BUILD rules for generating flatbuffer files in various languages.
+
+flatc_path = "//:flatc"
```

Comment:
Is it possible to make this compatible when flatbuffers is put in `//third_party/flatbuffers` instead of registered as a bazel repository? Maybe `//external:flatc`, but [`bind`](https://docs.bazel.build/versions/master/be/workspace.html#bind) is being deprecated.

Reply:
I put it in //third_party/flatbuffers in our repo.  The trick is to then write:

```local_repository(
    name = "com_github_google_flatbuffers",
    path = "third_party/flatbuffers",
)```

And then everything will just work.

From what I've been seeing for the last couple of years, the trend is to use ```local_repository``` (or one of the various other ways to pull in a repository) rather than ```//external``` and ```bind```.

Reply:
Thanks for the suggestion. I tried to avoid using sub-repo whenever a repo supports direct vendoring (used as `//third_party/...` instead of `@...`). Looks like the trend is not going that way :face_with_head_bandage:. Also it looks like sub-repo and `//third_party` are different but I don't find a canonical path for these sub-repos. We currently use `//vendor`.